mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -971,6 +984,26 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
         @BaseTransientBottomBar.Duration duration: Int = Snackbar.LENGTH_LONG,
         onClickListener: (() -> Unit)? = null
     ) {
+        showSnackbar(getString(messageResId), actionResId, tag, duration, onClickListener)
+    }
+
+    private fun showSnackbar(
```

Comment:
I wonder whether moving this to the base class would make sense, so it can be used from the other activities as well?

Reply:
Yes, that makes sense. I moved it to the abstract base activity.